nexus/executables/ntest

```diff
@@ -1680,13 +1830,82 @@ def exit_pass(msg=None):
     exit(0)
 #end def exit_pass
 
+# execute function
+from subprocess import Popen,PIPE
+def execute(command,verbose=False,skip=False):
+    out,err = '',''
+    if skip:
+        if verbose:
+            print 'Would have executed:\n  '+command
+        #end if
+    else:
+        if verbose:
+            print 'Executing:\n  '+command
+        #end if
+        out,err = Popen(command,shell=True,stdout=PIPE,stderr=PIPE,close_fds=True).communicate()
```

Comment:
The return value of the executed command should be checked for success or failure (or at least passed to the caller so that function can check)

Reply:
Return value added, see commit.